src/components/userPreferences/UserPreferences.jsx

```diff
@@ -0,0 +1,38 @@
+/*!
+* Copyright 2017 - 2020 by ChartIQ, Inc.
+* All rights reserved.
+*/
+import React, { useState } from "react";
+import ReactDOM from "react-dom"
+import "../../../assets/css/font-finance.css";
+import "../../../assets/css/finsemble.css";
+import "./userPreferences.css";
+import { UserPreferencesShell, LeftNav, ContentSection } from "@chartiq/finsemble-ui/react/components";
+
+// The preference menu entries. It is passed as a prop to the LeftNav component to generate the preference menu section names.
```

Comment:
I suggest JSDoc so this comment shows up on hover.
```suggestion
/** The preference menu entries. It is passed as a prop to the LeftNav component to generate the preference menu section names. */
```